docs/core/component.md

```diff
@@ -178,6 +178,16 @@ schema: {type: 'string'}  // default: ''
 schema: {type: 'vec3'}  // default: {x: 0, y: 0, z: 0}
 ```
 
+#### Specifying Possible Values
+
+You can also provide an array of acceptable values to use:
```

Comment:
Could this be misleading? The value range is not checked at run time. It's only a hint for the inspector to show the combo box widget. We should probably add a `#### Inspector` section explaining: the property type is used by the inspector to show the appropriate widget. `max` and `min` can be used to constrain the range of available values and `oneOf` to display a list of options.